GooglePlayInstant/Editor/AndroidManifest/AndroidManifestHelper.cs

```diff
@@ -210,6 +239,28 @@ private static string UpdateDefaultUrlElement(XElement mainActivity, Uri uri)
             return null;
         }
 
+        private static string UpdateDistributionModuleInstantElement(XElement manifestElement)
```

Comment:
Document the return value

Reply:
We don't usually document the private methods, but this seemed like a reasonable case for providing additional documentation. Done for this method and for all other methods in this class that return an error string.